core/src/main/scala/org.broadinstitute.dsde.workbench.leonardo/JsonCodec.scala

```diff
@@ -1,47 +1,119 @@
 package org.broadinstitute.dsde.workbench.leonardo
```

Comment:
The filename here looks weird - it looks like there's a dir named `org.broadinstitute.dsde.workbench.leonardo` as opposed to subdirs `org/broadinstitute/dsde/workbench/leonardo` (I checked out the branch to confirm).

Reply:
oh yea....my bad